src/constants/v2v3/ballotStrategies/index.ts

```diff
@@ -116,7 +116,3 @@ export function ballotStrategiesFn(network?: NetworkName): BallotStrategy[] {
     },
   ]
 }
-
-export const ballotStrategies = ballotStrategiesFn()
```

Comment:
trans strings above ^ were being evaluated outside the i18n provider scope. BUGZ